packages/graphql-plugin-auth/src/Neo4jGraphQLAuthJWTPlugin.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import jsonwebtoken from "jsonwebtoken";
+import Debug from "debug";
+import { DEBUG_PREFIX } from "./constants";
+
+const debug = Debug(DEBUG_PREFIX);
+
+export interface JWTPluginInput {
+    secret: jsonwebtoken.Secret;
+    noVerify?: boolean;
+    rolesPath?: string;
+}
+
+class Neo4jGraphQLAuthJWTPlugin {
+    private secret: jsonwebtoken.Secret;
+    private noVerify?: boolean;
+    rolesPath?: string;
```

Comment:
Does this have to be private?

```suggestion
    private rolesPath?: string;
```

Reply:
No, it's used by `@neo4j/graphql`.